apis/python/tests/test_registration_mappings.py

```diff
@@ -406,7 +309,7 @@ def test_isolated_anndata_mappings(obs_field_name, var_field_name):
 @pytest.mark.parametrize("var_field_name", ["var_id", "gene_id"])
 def test_isolated_h5ad_mappings(obs_field_name, var_field_name):
     h5ad1 = create_h5ad_canned(1, obs_field_name, var_field_name)
-    rd = registration.ExperimentAmbientLabelMapping.from_isolated_h5ad(
```

Comment:
Likewise please split out a PR which _only_ changes from `registration.ExperimentAmbientLabelMapping.from_isolated_h5ad` to your personal preference ExperimentAmbientLabelMapping.from_isolated_h5ad`, and necessary import mods.

You switching the style here to your personal preference here is a thing I don't disagree with. However, it clouds the picture while I'm trying to review to check for possible regressions (since you're modifying unit-test files, I can't rely on unit-test cases to _catch_ breakages/regressions here).

Splitting out these cosmetic PRs separately will help clear the picture and increase confidence in a solid review.

Reply:
I've removed these import tweaks from this PR